tests/drivers/rtc/rtc_api/src/main.c

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright 2022 Bjarki Arge Andreasen
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/ztest.h>
+
+ZTEST_SUITE(rtc_api, NULL, NULL, NULL, NULL, NULL);
```

Comment:
It might make sense to have the `after_fn` clear any alarms and make sure each test sets the alarm it requires.